### DISCUSSION TITLE
Integration with OKTA

### DISCUSSION THREAD

**Discussion by sudheeriilm** [microsoft/sample-app-aoai-chatGPT#678] [Q&A]
Hi Team, My business requirement is to integrate this app with OKTA. I am not able to find where the actual changes are needed.
Do I need to change the code in the Python file or need to change it to the React side?
Please guide also confirm me file name in which I need to change.

**Comment by rochaktarika123** [comment]
I am assuming that you are deploying this app on Azure. If you are, all you have to do is create an application credential in OKTA and in Azure under your webapp, provide the client and secret details in the Authentication section of your app. I have tested it and it works.
